test.js

```diff
@@ -1581,6 +1581,285 @@ test('errors', function (t) {
     })
 })
 
+test('token in cookie, with fastify-cookie parsing', function (t) {
+  t.plan(6)
+
+  const fastify = Fastify()
+  fastify.register(jwt, { secret: 'test', cookie: { cookieName: 'jwt' } })
+  fastify.register(require('fastify-cookie'))
+
+  fastify.post('/sign', function (request, reply) {
+    reply.jwtSign(request.body)
+      .then(function (token) {
+        return reply.send({ token })
+      })
+  })
+
+  fastify.get('/verify', function (request, reply) {
+    request.jwtVerify()
+      .then(function (decodedToken) {
+        return reply.send(decodedToken)
+      })
+      .catch(function (error) {
+        return reply.send(error)
+      })
+  })
+
+  fastify
+    .ready()
+    .then(function () {
```

Comment:
there is no need for this call to `ready()` or `then()`, it's done automatically at the first `inject()`.